src/systems/mecanum_drive/SpeedLimiter.cc

```diff
@@ -0,0 +1,200 @@
+/*********************************************************************
```

Comment:
We could use the new `ignition::math::SpeedLimiter` here, or just leave this one in for now and update later.

Reply:
good call

Reply:
9bbcceb